packages/augur-utils/src/generators.ts

```diff
@@ -0,0 +1,5 @@
+export function* chunkRange(start, end, chunk) {
+   for (;start < end;start+=chunk) {
```

Comment:
```suggestion
   for (;start =< end;start+=chunk) {
```

<img width="552" alt="image" src="https://user-images.githubusercontent.com/328965/78188550-49dc2e80-7425-11ea-925a-79314896f3c4.png">
The top has this patch, the second is as written here.